cmd/open.go

```diff
@@ -2,16 +2,27 @@ package cmd
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/railwayapp/cli/entity"
 )
 
 func (h *Handler) Open(ctx context.Context, req *entity.CommandRequest) error {
-	projectId, err := h.cfg.GetProject()
+	if len(req.Args) == 0 {
+		fmt.Println("Use railway open to open links to Railway from the CLI. Here's whats we have:")
```

Comment:
This doesn't always work if the project can't be found

<img width="626" alt="image" src="https://user-images.githubusercontent.com/3044853/96621908-24e7d400-1301-11eb-8597-000a0b6a358b.png">